webapp/src/components/modal/index.tsx

```diff
@@ -54,8 +55,7 @@ const Modal = ({show, onHide, showCloseIconInHeader = true, children, title, sub
                 onConfirm={onConfirm}
                 cancelBtnText={cancelBtnText}
                 confirmBtnText={confirmBtnText}
-                confirmDisabled={confirmDisabled}
```

Comment:
Why is this removed?

Reply:
this has been fixed here https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/40/files